docs/CommandClassDiagram.puml

```diff
@@ -0,0 +1,43 @@
+@startuml
+'https://plantuml.com/class-diagram
+package command {
+    abstract class Command
+    class DeleteCommand
+
+    Command <|-- DeleteCommand
+    Command <|-- ExitCommand
+    DeleteCommand <.. CommandResult
+    ExitCommand <.. CommandResult
+
+    abstract class Command {
+        +execute():CommandResult {abstract}
+    }
+
+    class DeleteCommand {
+    -moduleCode:String
+    -taskModule:String
+    -taskIndex:int
```

Comment:
should we include -result:String also since it is being returned in execute()

Reply:
I don't think we should as different commands seem to store the result in different locations. I think it is sufficiently shown in the CommandResult(result:String) constructor in the CommandResult class